mapie/metrics.py

```diff
@@ -1,7 +1,9 @@
-from typing import Optional, cast, Union
+from typing import Optional, cast, Union, Tuple
 
+import scipy
 import numpy as np
 from sklearn.utils.validation import check_array, column_or_1d
+from sklearn.utils import check_random_state
```

Comment:
Please add the imports in alphabetical order :)

Reply:
Done